src/component/preview.tsx

```diff
@@ -30,9 +31,13 @@ class PreviewApp extends React.Component<PreviewAppProps, PreviewAppState> {
 			// Ignored
 		}
 
+		const selectedElement: PageElement | undefined = this.props.store.getSelectedElement();
 		return (
 			<div>
```

Comment:
we could remove the wrapping div here i think

Reply:
sure but should not be done with this PR :)

Reply:
ah you're right :)